ci_framework/roles/hive/tasks/baremetal_deploy_cluster.yml

```diff
@@ -0,0 +1,99 @@
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+- name: Provisioner libvirtd access key
+  ansible.builtin.include_tasks:
+    file: oc_apply.yml
+  vars:
+    desc: "libvirtd SSH private key secret"
+    template_file: "templates/private_key.yml.j2"
+    dest_file: "libvirtd_ssh_private_key.yml"
+    secret_name: "{{ cifmw_hive_baremetal_cluster_name }}-libvirtd-access"
+    secret: "{{ lookup('file', '{{ cifmw_hive_oc_provisioner_user_key.filename }}') }}"
+
+- name: Enable Hive access to the provisioner host
+  ansible.builtin.include_tasks:
+    file: oc_apply.yml
+  vars:
+    desc: "Provisioner SSH private key secret"
+    template_file: templates/private_key.yml.j2
+    dest_file: ocp_ssh_private_key.yml
+    secret_name: "{{ cifmw_hive_baremetal_cluster_name }}-provisioner-access"
+    secret: "{{ lookup('file', '{{ cifmw_hive_oc_provisioner_root_key.filename }}' ) }}"
+
+- name: Load baremetal install config
+  ansible.builtin.include_vars:
+    file: "{{ cifmw_hive_baremetal_install_config }}"
+    name: bm_install_config
+
+- name: Add dynamic values into the bm install config
+  ansible.builtin.set_fact:
+    bm_install_config: "{{ bm_install_config | combine(item, recursive=true) }}"
+  loop:
+    - {'metadata': {'name': "{{ cifmw_hive_baremetal_cluster_name }}" }}
+    - {'platform': {'baremetal': {'libvirtURI': "qemu+ssh://root@{{ cifmw_hive_oc_provisioner }}/system" }}}
+    - {'platform': {'baremetal': {'externalMACAaddress': "{{ cifmw_hive_oc_provisioner_ext_mac }}" }}}
+    - {'sshKey': "{{ cifmw_hive_baremetal_ocp_access_key }}" }
+
+- name: Gather the base domain of the OCP cluster
+  ansible.builtin.set_fact:
+    cifmw_hive_ocp_base_domain: "{{ bm_install_config.baseDomain }}"
+
+- name: Add baremetal install config secret
+  ansible.builtin.include_tasks:
+    file: oc_apply.yml
+  vars:
+    desc: "install config secret"
+    template_file: templates/bm_install_config.yml.j2
+    dest_file: bm_install_config.yml
+    secret_name: "{{ cifmw_hive_baremetal_cluster_name }}-install-config"
+    install_config_yml: "{{ bm_install_config | to_nice_yaml(indent=2) }}"
+
+- name: OpenShift cluster image resource
+  ansible.builtin.include_tasks:
+    file: oc_apply.yml
+  vars:
+    desc: "ocp cluster image"
+    template_file: templates/ocp_image.yml.j2
+    dest_file: ocp_image.yml
+    image_name: "{{ cifmw_hive_baremetal_cluster_name }}"
+    image_uri: "{{ cifmw_hive_baremetal_ocp_image }}"
+    kubeconfig: "{{ cifmw_hive_kubeconfig }}"
```

Comment:
in ocp_image.yml.j2 you have:
`namespace: {{ cifmw_hive_namespace }}`
so you don't need this kubeconfig here?